tests/integration/targets/ios_lacp_interfaces/tests/cli/_remove_config.yaml

```diff
@@ -12,8 +12,12 @@
   register: result
   cisco.ios.ios_interfaces:
     config:
+      - name: Loopback1
       - name: Port-channel10
       - name: Port-channel11
       - name: Port-channel22
       - name: Port-channel40
+      - name: Vlan101
```

Comment:
Remove Vlans withing the tests where they are populated in this case it looks like l3_intf